src/index.ts

```diff
@@ -160,7 +160,72 @@ const transform = async ({
     exit(1);
   }
 
-  const functionsDir = resolve(".vercel/output/functions");
+  // RoutesManifest.version and RoutesManifest.basePath are the only fields accessed
+  interface RoutesManifest {
+    version: 3;
+    basePath: string;
+    pages404: boolean;
+    redirects: {
+      source: string;
+      destination: string;
+      basePath: boolean | undefined;
+      internal: boolean;
+      statusCode: number;
+      regex: string;
+    }[];
+    dynamicRoutes: {
+      page: string;
+      regex: string;
+      routeKeys: any; // object of dynamic parameters;
+      namedRegex: string;
+    };
+    staticRoutes: {
+      page: string;
+      regex: string;
+      routeKeys: any; // object of dynamic parameters;
+      namedRegex: string;
+    }[];
+    rsc: {
+      header: string;
+      varyHeader: string;
+    };
+    headers: unknown;
+    dataRoutes: unknown;
+    rewrites: unknown;
```

Comment:
These types could use some work if requested, though may not be necessary as they're not used in this package at all...